### PR TITLE
#13 - btoa error on node 14

### PR DESCRIPTION
**Pull request by djavorek**
## Description of Changes
- 🍪Using **Buffer.from calls to encode/decode binary strings**, instead of btoa/atob calls. (To support node 14) 
  - Added the currently supported _npm_ and _node_ versions into the README. We can add more to the list as we test them.
- 🛠Configured **Jest** to find and run our tests
  - Additional configuration to be able to test Svelte components
    - It is building the whole app, in a way Jest can handle it (**Babel transpiling, Svelte preprocessing**)
    -  📜Initial guide: https://el3um4s.medium.com/how-to-test-sveltekit-app-with-jest-848afa8edbc7
- ✅ Added Jest **test for Cookie generation** and the first trivial **Svelte component test** 
  - More to come in my next PR..

Issue: https://github.com/szattila98/veryrezsi/issues/13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+
+- npm v7 or higher
+  - 8.4.1 ✅
+- node v14 or higher
+  - v16.14.0 ✅
+
+## Development
+- `cd ./veryrezsi/client`
+- `npm i`
+- `npm run dev`
```
Comment (szattila98) [inline README.md:16]: good stuff, I was planning on writing a readme, thanks, I will write the server counterpart then

File: client/babel.config.cjs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	presets: [
+		[
+			'@babel/preset-env',
```
Comment (szattila98) [inline client/babel.config.cjs:4]: ༼つ ◕_◕ ༽つ TRANSPILE ༼ つ ◕_◕ ༽つ﻿

File: client/jest-setup.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+import '@testing-library/jest-dom';
```
Comment (szattila98) [inline client/jest-setup.ts:1]: A one line setup?

**Review by szattila98 — APPROVED** [review]
Nice work, you widened the scope a bit - but useful stuff nonetheless.

File: README.md (hunk 1 above)
Comment (djavorek) [reply README.md:16]: Yep, it was just the easy one.

File: client/jest-setup.ts (hunk 3 above)
Comment (djavorek) [reply client/jest-setup.ts:1]: Well, this just runs before every test file.
Also check out:  `@jest-environment jsdom` comment below in the test. That is more than just a comment, as it sets up jsdom for that file specifically :D

Comment (szattila98) [reply client/jest-setup.ts:1]: Oh I see.
It is always confusing for me when comments are more than just comments.

**Pull request merged**